pom.xml

```diff
@@ -286,6 +290,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Can we make a short comment why this is necessary? I'm afraid that someone will remove it and will break tests silently. :-)